app/graphql/types/rule.rb

```diff
@@ -28,7 +28,10 @@ def compliant(system_id:)
     def references(lookahead:)
       selected_columns = lookahead.selections.map(&:name) &
                          ::RuleReference.column_names.map(&:to_sym)
-      object.references.pluck(selected_columns).map do |reference|
+      qualified_columns = selected_columns.map { |c| "rule_references.#{c}" }
+      object.references.pluck(
+        "DISTINCT #{qualified_columns.join(', ')}"
```

Comment:
@akofink I had to add the qualified name here, because `.distinct.pluck(selected_columns)` would not work, unless I qualify the columns, and if I do that, then `selected_columns.zip([reference].flatten).to_h` returns something that is not expected by the frontend.